Makefile

```diff
@@ -100,99 +139,52 @@ test-mutation: verify-no-changes fetch-mutation ## Run mutation tests in manual
 test-mutation-ci: fetch-mutation ## Run mutation tests as part of auto build process.
 	./hack/test-mutation.sh
 
-# Build manager binary
-manager: generate fmt vet
+##@ Build
+
+manager: generate fmt vet ## Build manager binary
```

Comment:
In other operators we are using the word `build` for this commend. I have no strong opinion which one is better.